README.md

```diff
@@ -86,20 +85,20 @@ This section should be referred to for next steps when any indicator reports a f
-| S0034        | Determines if PE software is more than 1 year old                   | Please refer to https://puppet.com/docs/pe/2021.4/upgrading_pe.html for instructions on how to upgrade your PE instance                                                                                                                | If you need advice on upgrading PE  - Please raise a ticket with the version of PE you are currently running and the version you would like to upgrade to (this could be the LTS or STS version of PE). |
+| S0034        | Determines if PE deployment has not been upgraded in the last year                   | Please refer to https://puppet.com/docs/pe/2021.4/upgrading_pe.html for instructions on how to upgrade your PE instance                                                                                                                | If you need advice on upgrading PE  - Please raise a ticket with the version of PE you are currently running and the version you would like to upgrade to (this could be the LTS or STS version of PE). |
 | S0036        | Determines if max-queued-requests is set above 150                       | The maximum value for `jruby_puppet_max_queued_requests` is 150                (<https://support.puppet.com/hc/en-us/articles/115003769433>)                                                                      | If you are unable, or encounter an error when changing the value of `jruby_puppet_max_queued_requests`, raise a support case quoting reference S0036 and any errors output when attempting to change the setting
-| S0039        | If puppetserver has a non zero queue-limit-hit-rate, Puppetserver will send 503 error messages. | Please refer the Knowledge article https://support.puppet.com/hc/en-us/articles/115003769433                     | Raise a support case quoting reference S0039 together with the [Support Script](https://puppet.com/docs/pe/2021.4/getting_support_for_pe.html#pe_support_script) output from the primary node
-| S0040        | Determines if you collecting system metrics                    | Refer to the following documentation when setting up system metrics: <https://forge.puppet.com/modules/puppetlabs/puppet_metrics_collector#installation>                                                                     | Raise a support request, if after system metrics are configured and you do not see any files in /var/log/sa or if /var/log/sa directory does not exist.                                                 |
+| S0039        | Determines if Puppetserver has reached its queue-limit-hit-rate,and is sending messages to agents. | Please refer the following Knowledgebase article https://support.puppet.com/hc/en-us/articles/115003769433                     | Raise a support case quoting reference S0039 together with the [Support Script](https://puppet.com/docs/pe/2021.4/getting_support_for_pe.html#pe_support_script) output from the Primary node
```

Comment:
Should it be:
| S0039        | Determines if Puppetserver has reached its queue-limit-hit-rate, and is sending messages to agents. | Please refer to the following Knowledgebase article https://support.puppet.com/hc/en-us/articles/115003769433                     | Raise a support case quoting reference S0039 together with the [Support Script](https://puppet.com/docs/pe/2021.4/getting_support_for_pe.html#pe_support_script) output from the Primary node

Also are "the messages" 503's as per the previous description or are there other messages